support/mender-inventory-provides

```diff
@@ -12,4 +12,8 @@ set -e
 # entries.
 grep '^device_type=' /var/lib/mender/device_type
 
+# Submit client version
+# TODO: MEN-6840: Remove when developing the built-in version
+echo mender_client_version=`/usr/bin/mender-update --version`
```

Comment:
Q: Should we skip `/usr/bin` and rely on the PATH?

Reply:
Beware of the comment in the previous block! Put this one above it.

Reply:
> Q: Should we skip `/usr/bin` and rely on the PATH?

Few lines above we are already using `/usr/bin/`. I am not comfortable changing it, really. (Also for this specific case it will be reverted, but your point stands valid for the pre-existing code)

Reply:
> Beware of the comment in the previous block! Put this one above it.

Thank you!